modules/osdk-project-staffolding-layout.adoc

```diff
@@ -8,36 +8,41 @@
 The `operator-sdk` CLI generates a number of packages for each Operator project. The following sections describes a basic rundown of each generated file and directory.
 
 [id="osdk-project-scaffolding-layout-go_{context}"]
-== Go-based projects
+== Ansible-based projects
 
-Go-based Operator projects (the default type) generated using the `operator-sdk new` command contain the following directories and files:
+Ansible-based Operator projects generated using the `operator-sdk new --type ansible` command contain the following directories and files:
 
 [options="header",cols="1,2"]
 |===
 
 |File/folders |Purpose
 
-|`cmd/`
-|Contains `manager/main.go` file, which is the main program of the Operator. This instantiates a new manager which registers all custom resource defintitions (CRDs) under `pkg/apis/` and starts all controllers under `pkg/controllers/`.
+|`molecule/`
+|Contains the files that are used for testing the ansible roles.
 
-|`pkg/apis/`
-|Contains the directory tree that defines the APIs of the CRDs. Users are expected to edit the `pkg/apis/<group>/<version>/<kind>_types.go` files to define the API for each resource type and import these packages in their controllers to watch for these resource types.
+// |`molecule/cluster`
+// |Contains files that is used to run an end-to-end test of your operator against an existing cluster. The operator image needs to be available to the cluster.
```

Comment:
```suggestion
// |Contains files that are used to run an end-to-end test of your Operator against an existing cluster. The Operator image needs to be available to the cluster.
```

Reply:
`The Operator image needs to be available to the cluster.`
* You discuss two Operators, and it's not immediately clear which Operator this sentence refers to.